mono/mcs/class/System.Windows.Forms/System.Windows.Forms/ComboBox.Mac.cs

```diff
@@ -1534,6 +1533,8 @@ public ComboBoxImp(ComboBox owner)
 
 				if (owner.selected_index >= 0 && owner.selected_index < combo.Count)
 					combo.SelectItem(owner.selected_index);
+					
+				combo.SelectionChanged += ComboSelectionChanged;
```

Comment:
I assume the only change is moving this line a but further down to avoid firing the events during initialization, right? If that's the case then it looks good to me.

(The other white-space changes (eg. deleting empty spaces at end of line) make the diff a bit harder to read. They should be done though.)

Reply:
Thanks!

Reply:
You absolutely right it prevent firing event during initialization.